Class-Examples/XML/emilyAssignment3.xml

```diff
@@ -0,0 +1,202 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<root>
+    <para num="1">Knowing how to change a tire is a necessary skill for all drivers. If you rely on
+        a cell phone to save you in a roadside emergency, there’s always that chance you will forget
+        to charge it, be out of range, or leave it at home. Flat tires can happen anywhere, and a
+        cell phone is no substitute for knowing how to change a flat tire.</para>
+    <para num="2">Thankfully, changing a tire isn’t all that hard! Just adhere to the following
+        guidelines to be prepared in case you have a flat.</para>
+    <para num="3">
+        <heading>ITEMS YOU'LL NEED TO FIX A FLAT TIRE</heading>
+        <subheading>These items should have come with your vehicle:</subheading>
+        <item xml:id="jack">Jack</item>
+        <item xml:id="lWrentch">Lug wrench</item>
+        <item xmlid="spare">Fully inflated spare tire</item>
+        <item xml:id="manual">Vehicle owner’s manual</item>
+    </para>
+    <para num="4">
+        <body>If you have misplaced any of these items, or if your car did not come with these
+            items, you should purchase new ones right away. And be sure you’re regularly inflating
+            the spare tire to your vehicle manufacturer’s recommended PSI. You should check the
```

Comment:
So **here** where the spare tire is mentioned on line 20, we have an opportunity to insert some markup that holds a pointer back to the xml:id you coded. You might do something like this:

```xml
    <body>If you have misplaced.....be sure you're regularly inflating 
         <ref target="#spare">the spare tire</ref> to your vehicle manufacturer's recommended PSI...
      </body>
```

Here I'm inserting an element with an attribute designed to point to the "canonical" reference where the instructions first identify the item. The attribute value for `@target` isn't quite the same as the `@xml:id`: I put a hashtag in front of it to make it distinct. That way I can use it as many times as I want to, just using the value to refer back to the distinct item in the list.